### DISCUSSION TITLE
When hovering over songs in the song market, besides showing song info, show dominant instruments

### DISCUSSION THREAD

**Discussion by marceloc10uy** [ilpersi/Popmundo-Utils#3] [Ideas]
Basically what it says in the title, use the already existing module to hover over songs and get its data, but in the song market add the dominant instruments so that i can check more quickly if the song has the instruments I need the song to have.

**Comment by ilpersi** [comment]
This is included in 0.5.2. Check it out. 👍🏻

**Comment by ilpersi** [comment]
@marceloc10uy is this working as expected for you? :)

**Comment by marceloc10uy** [comment]
Works great.
